components/app.js

```diff
@@ -0,0 +1,28 @@
+class App {
+  handleGetGradesError(error) {
+    console.error(error);
+  }
+  handleGetGradesSuccess(grades) {
+    this.gradeTable.updateGrades(grades);
+  }
+  constructor() {
+    this.handleGetGradesError = this.handleGetGradesError.bind(this);
+    this.handleGetGradesSuccess = this.handleGetGradesSuccess.bind(this);
+    this.gradeTable = gradeTable;
+  }
+  getGrades() {
+    $.ajax({
+      url: "https://sgt.lfzprototypes.com/api/grades",
+      method: "GET",
+      data: "none",
```

Comment:
`data; "none"` sends a request body containing `none`. If you don't wish to send a request body, don't supply a `.data` property.